docs/create-sharded-table.rst

```diff
@@ -0,0 +1,120 @@
+.. _create-sharded-table:
+
+====================
+Create sharded table
+====================
+
+One core concept CrateDB uses to distribute data across a cluster is sharding. 
+CrateDB splits every table into a configured number of shards, which are 
+distributed evenly across the cluster. You can think of shards as sub-tables 
+with corresponding sub-indexes. If we create a table like the following:
+
+.. code-block:: psql
+
+    CREATE TABLE first_table (
+        ts TIMESTAMP,
+        val DOUBLE PRECISION
+    );
+
+The table is by default is split into 4 shards on a single node cluster. You 
+can check this by running:
```

Comment:
```suggestion
The table is by default is split into several shards. How much exactly depends on the number of nodes in the cluster. You can check this by running:
```

Not sure if it's better - but rephrasing it would make the document more resilient against a change of the default value in CrateDB